tests/typecheck/lostno/LostNo.java

```diff
@@ -1,16 +1,15 @@
-package typecheck.lostno;
+package universe.lostno;
```

Comment:
Is there a reason for these package changes? They don't match the directory names anymore.

Reply:
I have noted it down. I will resolve these issues in a separate PR.